tests/roles/stop_openstack_services/templates/systemctl_disable.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Try to stop a service , fail only when the service is present and doesn't stop correctly.
```

Comment:
This comment seems to be copy-paste for systemctl_stop.

Reply:
ack , updated he comment